src/main/java/com/esri/samples/map/generate_offline_map_overrides/GenerateOfflineMapOverridesController.java

```diff
@@ -110,6 +107,9 @@ private void initialize() {
       updateDownloadArea();
```

Comment:
The download area graphic is still not always appearing after log in: I've had a play around with the code too but can't figure out a way to make it always appear. Happens once every 3-4 attempts.